src/screens/Login/Login.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import {
+  TextInput,
+  TouchableHighlight,
+  Text,
+  Image,
+  KeyboardAvoidingView
+} from "react-native";
+import R from "ramda";
+
+import { styles } from "./styles";
+
+const logoImg = require("../../assets/logo.png");
```

Comment:
I believe it would be nice to set up alias for content of src folder so that import could be just "assets/logo.png", it's just my thoughts in advance, that shouldn't block this pr from merging

Reply:
Agreed!